static/js/util/validation_test.js

```diff
@@ -172,6 +166,25 @@ describe('Profile validation functions', () => {
       assert.deepEqual(errors, employmentValidation(profile));
     });
 
+    it('should reject an end date in the future', () => {
+      sandbox.useFakeTimers(moment('2016-10-01').valueOf());
+      let expectation = {
+        work_history: [, // eslint-disable-line no-sparse-arrays
+          { end_date: 'End date cannot be in the future' }
+        ]
+      };
+      let profile = _.cloneDeep(USER_PROFILE_RESPONSE);
+      profile.work_history[1].end_date = moment().add(1, 'month');
```

Comment:
We should format the moment object to a string here and below. Can you also make this change to other tests in this file?